Networking

```diff
@@ -186,4 +186,24 @@ listening on enp0s3, link-type EN10MB (Ethernet), capture size 262144 bytes
 33 packets received by filter
 0 packets dropped by kernel
 
+_______________________________
 
+First window.			# For more strict we can add ip address of the host, also we can add the option of display output in hexadecimal format
+[Dima@test ~]$
+sudo tcpdump -A -X -i -I enp0s8 host 192.168.56.202
```

Comment:
Will it catch only packets with your curl command (to yandex.ru) or it will catch all packets related to host 192.168.56.202 and interface enp0s8 ?

Reply:
It will catch all packets related to the host. What exactly do you mean when asked about the command strict?

Reply:
It should catch **only** packets related to your request